### DISCUSSION TITLE
bug: reduce the Pop-up box show time

### DISCUSSION THREAD

**Discussion by GladioFeng** [LazyVim/LazyVim#1379] [Q&A]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.9.1

### Operating system/version

ubuntu16.04

### Describe the bug

![image](https://github.com/LazyVim/LazyVim/assets/100987925/4f1c5158-a164-462f-9743-142f332f3959)
**How to reduce the Pop-up box show time, it will show 5 second or more time!** 

### Steps To Reproduce

cd ~/.config/nvim/lua/config
vim options.lua

### Expected Behavior

Pop-up box show 100ms 

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  "folke/LazyVim",
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

**Comment by abeldekat** [comment]
Hello @GladioFeng,

>How to reduce the Pop-up box show time, it will show 5 second or more time!

This is not a bug in LazyVim.
Could you close this issue and ask your question in the [discussions](https://github.com/LazyVim/LazyVim/discussions) section?

Best regards!

**Comment by abeldekat** [comment]
Hello @GladioFeng,

In your config, you could add:
```lua
  {
    "rcarriga/nvim-notify",
     opts = {
       timeout = 100,
     },
  }
```

The default timeout is 3000ms.

LazyVim also has an option to dismiss the notification: "leader u n"
